hawkular-bus-common/src/main/java/org/hawkular/bus/common/consumer/AbstractBasicMessageListener.java

```diff
@@ -131,49 +130,31 @@ public void setConsumerConnectionContext(ConsumerConnectionContext consumerConne
         BasicMessageWithExtraData<T> retVal;
         try {
             Class<T> basicMessageClass = null;
-
-            // If a basic message class name was provided to us in the header, we will try our best to use that
-            // unless a subclass wants to substitute another class for it.
-            String basicMessageClassName = message.getStringProperty(MessageProcessor.HEADER_BASIC_MESSAGE_CLASS);
-            if (basicMessageClassName != null) {
-                String desired = convertReceivedMessageClassNameToDesiredMessageClassName(basicMessageClassName);
```

Comment:
you gotta put this back - me and jay purposefully made these changes to support things alerts/events needs. This change reverts back to old code that will now break alerts/events.